_docs/install-and-upgrade.md

```diff
@@ -106,7 +119,7 @@ thelounge start
 The Lounge is now up and running **in private mode** at <http://localhost:9000>.
 
 Read more about how to use The Lounge from the command line in
-[the CLI usage section](/docs/cli_usage.html).
+[the usage section](/docs/usage.html).
```

Comment:
I _think_ I did mean "CLI usage section", but I'm not 100% at this point. Let me get back to this work and I'll confirm or not.

Reply:
CLI usage page doesn't exist, but the wording can be left with "CLI usage"